db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/amazon_check_pre_retirement.rb

```diff
@@ -5,22 +5,24 @@
 
 # Get vm from root object
 vm = $evm.root['vm']
+ems = vm.ext_management_system
```

Comment:
The next line will check if vm = nil or ems = nil, the process will log a message and abort.